kong/types.go

```diff
@@ -157,7 +157,7 @@ type PassiveHealthcheck struct {
 type Healthcheck struct {
```

Comment:
This PR introduces changes of internal representation that are expected to preserve the interface. This needs to be tested.

Please implement tests checking that both JSON and YAML marshalling and unmarshalling correctly handle at least the following scenarios:
- field present
- field present but `0`
- field not present